src/components/pages/AppOverview/ChangeDescription.tsx

```diff
@@ -57,10 +57,18 @@ export default function ChangeDescription() {
 
   const defaultValues = useMemo(() => {
     return {
-      longDescriptionEN: description?.[0]?.longDescription ?? '',
-      longDescriptionDE: description?.[1]?.longDescription ?? '',
-      shortDescriptionEN: description?.[0]?.shortDescription ?? '',
-      shortDescriptionDE: description?.[1]?.shortDescription ?? '',
+      longDescriptionEN:
```

Comment:
we can reduce number of find methods to 2. define two const values and read them in the return statement.
const en = a.find((b) => b.languageCode === 'en')
finally -> en.longDescription ?? ''